src/consensus/versionbits.cpp

```diff
@@ -0,0 +1,398 @@
+// Copyright (c) 2009-2015 Eric Lombrozo, The Bitcoin Core developers
```

Comment:
Should stick to the standard header used across the codebase. Copyright always belongs to the author by default, there is no need to explicitly add it otherwise it gets really messy really quickly.